pyInterface/python_compiler.py

```diff
@@ -13,4 +13,5 @@
 
 	os.system('rm -rf ' + destdir + '/*')
 	os.system('cd ' + indir + '; for dir in `find * -type d`; do mkdir ' + destdir + '/$dir; done; cd - > /dev/null')
-	os.system('cd ' + indir + '; for pycfile in `find * -name "*.pyc"`; do mv $pycfile ' +  destdir + '/$pycfile; done; cd - > /dev/null')
+	os.system('cd ' + indir + '; for pycfile in `find . -name "*.pyc"`; do mv $pycfile ' +  destdir + '/$pycfile; done; cd - > /dev/null')
+	os.system('cd ' + indir + '; for pyfile in `find . -name "*.py"`; do cp $pyfile ' + destdir + '/$pyfile; done; cd - > /dev/null')
```

Comment:
As the package folder has the wrong name ('package' insead of 'pyRootPwa'), this does not work. 
I do not know if pyLint checks the full package if you give him the __init__ file. 
Furthermore, by putting the .py files into the build directory, eclipse started to autocomplete the pyRootPwa package. 
Therefore I would prefer to have the .py files twice. In C++ you also have your header files twice.